### DISCUSSION TITLE
Preferences > I/O: how about renaming it to "Audio/MIDI"?

### DISCUSSION THREAD

**Discussion by cbjeukendrup** [musescore/MuseScore#14584] [Ideas]
Maybe it's just me, but I think the term "I/O" might be difficult to understand if you don't already know what it means. I would like to propose to rename the I/O tab in Preferences to something like "Audio/MIDI", which (I think) everyone will understand the first time they see it. Why using difficult jargon if a simple, more descriptive term works too? :)

**Comment by Tem3dy** [comment]
I think it would be better named as just "Audio" or something but yeah I agree.. or Input / Output instead of just I/O

**Reply by shoogle** [reply]
Or even "Audio I/O". But I think "Audio/MIDI" is best because it's not 100% clear that MIDI options will come under "Audio" or "Audio I/O".

**Comment by SteveBlower** [comment]
The terms input and output _are_ too general.  In this case they really only refer to midi input and midi and audio output.  They don't cover other inputs - e.g. PC keyboard, mouse; or outputs - e.g. printer, export to pdf/mp3 etc.  Audio/Midi seems to be about right, or perhaps restrict the scope even further by calling it Audio/Midi devices.
